lib/tasks/user/generate_token.rake

```diff
@@ -0,0 +1,7 @@
+desc "Generates Token"
+namespace :user do
+  task :generate_token, [:email] => :environment do |t, args|
+    user = User.find_by(email: args[:email])
+    user.regenerate_auth_token
```

Comment:
use the public method you created in `user.rb`

Reply:
done